src/components/logo/logo.tsx

```diff
@@ -126,7 +126,7 @@ class Logo extends Component<any, any> {
           onClick={this._handleOnClick}
           className={style.controlButton}
           href={this.state.urlLink}
-          aria-label={props.logoText}
+          aria-label={props.config.text ? props.config.text : "Logo"}
```

Comment:
first of all, please update the PR description.

second, props.logoText is localized, while the hardcoded "Logo" is not. did you mean to do something like aria-label={props.config.text || props.logoText} ?

Reply:
Yes, you are right, it's better this way

Reply:
Done